tracer/src/Datadog.Trace.Tools.dd_dotnet/Utils.cs

```diff
@@ -0,0 +1,181 @@
+// <copyright file="Utils.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.InteropServices;
+using Spectre.Console;
+
+namespace Datadog.Trace.Tools.dd_dotnet;
+
+internal class Utils
+{
+    public const string Profilerid = "{846F5F1C-F9AE-4B07-969E-05C26BC060D8}";
+
+    public static bool IsAlpine()
+    {
+        try
+        {
+            if (File.Exists("/etc/os-release"))
```

Comment:
Given this is only native AOT compiled, we could probably "bake in" whether the executable is alpine or not. Because we have a different binary for alpine right? (Which will only execute if we're on alpine!)

In fact, that makes me wonder... should we use `#ifdef` for windows/linux etc instead of the normal `RuntimeInformation`, that way we can force the linker to trim stuff for each platform 🤔 e.g. we don't need any of the IIS stuff on linux!

Reply:
I checked, removing the IIS bit saves ~30KB on the compressed binary. Given that there is no predefined `#if` variable to check the OS, it adds a bit of complexity. I'm not sure it's worth it. I can try in another PR to see if we can save more by applying this everywhere.